apps/accounts/migrations/0006_alter_user_about_alter_user_img_url.py

```
@@ -0,0 +1,23 @@
+# Generated by Django 5.1.6 on 2025-02-27 13:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounts', '0005_alter_user_is_active'),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)